network-api/networkapi/settings.py

```diff
@@ -490,6 +490,8 @@
     'NETWORK_SITE_URL': env('NETWORK_SITE_URL'),
     'TARGET_DOMAIN': env('TARGET_DOMAIN'),
     'SHOW_TAKEOVER': env('SHOW_TAKEOVER'),
+    'FELLOWSHIPS_PROGRAM_INFO_TYPES': env('FELLOWSHIPS_PROGRAM_INFO_TYPES').split(','),
```

Comment:
If you tell environ about these vars, and declare them as `list` types, you can avoid having to manually split them.

Here's an example of how we load the ALLOWED_HOSTS variable: https://github.com/mozilla/foundation.mozilla.org/blob/c1a9b27e69ee6c80107cc8d326f982f900dea140/network-api/networkapi/settings.py#L24-L26